File: src/main/kotlin/cn/xxstudy/api/pojo/Category.java
package cn.xxstudy.api.pojo;

import lombok.Data;

import java.time.LocalDateTime;

/**
 * @date: 2024/7/8 18:27
 * @author: TangRen
 * @remark:
 */
@Data
public class Category {
    private int id;
    public String categoryName;
    private String categoryAlias;
    public int createUser;
    private LocalDateTime createTime;
    private LocalDateTime updateTime;

    public Category(String categoryName, String categoryAlias) {
        this.categoryName = categoryName;
        this.categoryAlias = categoryAlias;
    }
}

File: src/main/kotlin/cn/xxstudy/api/pojo/Category.kt
package cn.xxstudy.api.pojo

import java.time.LocalDateTime

/**
 * @date: 2024/7/9 16:41
 * @author: TangRen
 * @remark:
 */
data class Category(
    var id: Int? = null,
    var categoryName: String? = null,
    var categoryAlias: String? = null,
    var createUser: Int? = null,
    var createTime: LocalDateTime? = null,
    var updateTime: LocalDateTime? = null
)